homeassistant/components/envisalink/__init__.py

```diff
@@ -210,6 +217,11 @@ async def handle_custom_function(call: ServiceCall) -> None:
 
     _LOGGER.info("Start envisalink")
     controller.start()
+    if panel_type == PANEL_TYPE_DSC and not create_zone_bypass_switches:
+        # Temporary hack to prevent the underlying pyenvisalink package from sending periodic *1# commands to the EVL
+        _LOGGER.info("Disabling zone bypass refresh task")
+        # pylint: disable=W0212
```

Comment:
I have a PR open for the pyenvisalink package (https://github.com/Cinntax/pyenvisalink/pull/21) which contains a proper corrective fix for the underlying issue.  Unfortunately this issue is quite critical for some users and so this specific code provides a temporary solution until the package itself can be updated.  If that happens before this PR is complete then I will make the necessary updates here but I wanted to provide a quick option for users who are currently affected.